src/EditorFeatures/CSharpTest/CodeActions/InvertIf/InvertIfTests.Elseless.cs

```diff
@@ -0,0 +1,602 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings.InvertIf
+{
+    public partial class InvertIfTests
+    {
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless01()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        foreach (var item in list)
+        {
+            [||]if (!c)
+                return;
+            f();
```

Comment:
do you have an example wehre this is written as  ```f(); throw/return/break;```?  I'm curious how it translates.  Because the 'if' will not reach the end, you won't need an else.

Reply:
>Because the 'if' will not reach the end, you won't need an else.

Imo that should be a part of another fix i.e. fading `else`. Perhaps we can make it as part of the simplifier?

Reply:
(bydesign)